tests/integration/targets/activation/tasks/main.yml

```diff
@@ -189,6 +197,109 @@
     - name: List all the rulebook activations
       ansible.eda.activation_info:
 
+    - name: Delete rulebook activation
+      ansible.eda.activation:
+        name: "{{ activation_name }}"
+        state: absent
+
+    - name: Get information about the rulebook activation
+      ansible.eda.activation_info:
+        name: "{{ activation_name }}"
+
+    # Test event_streams option
+    - name: Create a new credential
+      ansible.eda.credential:
+        state: present
+        name: "{{ credential_name_basic }}"
+        description: "This is a test credential"
+        credential_type_name: "Basic Event Stream"
+        inputs:
+          username: "test"
+          password: "test"
+        organization_name: Default
+      register: _result
+
+    - name: Create an event stream
+      ansible.eda.event_stream:
+        state: present
+        name: "{{ event_stream_name }}"
+        credential_name: "{{ credential_name_basic }}"
+        organization_name: Default
+      register: _result
+
+    - name: Get information about a rulebook
+      ansible.eda.rulebook_info:
+        name: "{{ rulebook_name }}"
+        project_name: "{{ project_name }}"
+      register: _result_rulebook_info
+
+    - name: Create a rulebook activation (wrong source_name)
+      ansible.eda.activation:
+        name: "{{ activation_name_wrong_source }}"
+        description: "Example Activation description"
+        project_name: "{{ project_name }}"
+        rulebook_name: "{{ rulebook_name }}"
+        decision_environment_name: "{{ decision_env_name }}"
+        enabled: False
+        awx_token_name: "{{ awx_token_name }}"
+        organization_name: Default
+        event_streams:
+          - event_stream: "{{ event_stream_name }}"
+            source_name: "Test source name"
+      register: _result
+      ignore_errors: true
+
+    - name: Check rulebook activation creation
+      assert:
+        that:
+          - not _result.changed
+          - '"The specified source_name Test source name does not exist." in _result.msg'
+
+    - name: Create a rulebook activation
+      ansible.eda.activation:
+        name: "{{ activation_name_source_index }}"
+        description: "Example Activation description"
+        project_name: "{{ project_name }}"
+        rulebook_name: "{{ rulebook_name }}"
+        decision_environment_name: "{{ decision_env_name }}"
+        enabled: False
+        awx_token_name: "{{ awx_token_name }}"
+        organization_name: Default
+        event_streams:
+          - event_stream: "{{ event_stream_name }}"
+            source_index: 0
+      register: _result
+
+    - name: Check rulebook activation creation
```

Comment:
Would be good if we can retrieve the activation and ensure it is mapped to the event stream, as it is now that should fail because the event stream was never mapped since the api would ignore the wrong "event_streams" parameter sent at the creation.